docs/articles/documentation/reference/test-api/requesthook/README.md

```diff
@@ -103,3 +103,35 @@ async onResponse (event) {
     throw new Error('Not implemented');
 }
 ```
+
+{% include notes-warnings/requesthook-onresponse-no-stub-headers.md href='#change-or-delete-response-headers' %}
+
+### Change or Delete Response Headers
+
+You can use the `RequestHook._onConfigureResponse()` method to set the header value or remove existing header, effectively mocking part of the response.
```

Comment:
My point was: I - the user - do not care that I can use X for Y purpose, unless: a) I already use X or b) it's the best way to achieve Y.

To give you an example: I can - in theory - smash roaches with an antique clock. However, if you say to me that "You can use an antique clock to smash roaches" I'd be confused because a) I don't have an antique clock and b) It sounds suboptimal c) I don't think I have a roach problem 

"You can use..." sounds like: a) you can do other things with the onConfigureResponse method. *If you can - please describe them* b) you can mock request headers with other methods. *If you can - mention them* c) I already want to change headers *Why would I want to change headers?*

```suggestion
The `RequestHook._onConfigureResponse()` method sets and remove headers, effectively mocking part of the response.
```

As of now, I'm not 100% sure this file *as a whole* - not your fault - does a good job of explaining what `RequestHooks`are, and why I would ever need one... 

We should always try to imagine someone reading everything out of context - as if they followed an anchor link on Google - and try to give them a complete enough picture